client/src/components/Filter.jsx

```diff
@@ -21,11 +22,13 @@ const Filter = ({ filter, setFilter }) => {
   }, []);
 
   return (
-    <>
+    <FilterWrapper>
       {dropdownFilter.hold_colors.length > 0 && (
         <>
-          <label for="hold_color">Choose a hold color:</label>
-          <select
+          <label className="sr-only" for="hold_color">
```

Comment:
We talked about it yesterday, but I just want to remark again that I would prefer a separate styled component for your `sr-only` stylings that you import where you need it. :)

Reply:
@mbosselmann 
Hi Mareike,
That's on the refactor list. I promise I will take care of it... But thinking of it I have a question which might sound silly but I can't get my head around it. How exactly do I separate it into a component. In styled.components I always have to define the HTML element which it should return, but I guess in this case this could differ. So how do I do it when I want to have this flexibility about the actual element? Can you maybe give me a hint?

Reply:
Sure! You can work with the attribute `as`. An example: your styled component `SrOnly` is based on a `span` element, but you want to use it for a `label`. Then you can use it like this `<SrOnly as="label" ...></SrOnly>`. :)

Reply:
@mbosselmann I implemented it now! :-)